core/src/main/java/io/questdb/cutlass/http/processors/JsonQueryProcessorState.java

```diff
@@ -506,11 +507,9 @@ private static void putUuidValue(HttpChunkedResponse response, Record rec, int c
 
     private boolean addColumnToOutput(
             RecordMetadata metadata,
-            CharSequence columnNames,
-            int start,
-            int hi
+            CharSequence columnNames
```

Comment:
parameter name consistency issue: what's the connection between `columnsQueryParameter` passed in (it is already accessible as class member) and plural "columnNames". We need to keep method signatures such that whoever is next working on the code does not have to reverse engineer the method to understand meaning of the signature

Reply:
it looks like we're using column name sink and here there is singular column (after reverse engineering)